src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java

```diff
@@ -456,7 +456,8 @@ private void routeAgencyDocument(Agency agency, Agency oldAgency) {
 		agencyDoc.setNewMaintainableObject(agencyMaintainable);
 		try {
 			documentService.saveDocument(agencyDoc);
-			agencyDoc.getDocumentHeader().getWorkflowDocument().route("Automatically created and routed");
+			GlobalVariables.getMessageMap().clearErrorMessages();
```

Comment:
Further above, workflow exceptions are caught.  Should those exceptions prevent the document from routing?  If so, clearing the error messages may prevent that.

Reply:
I think that the Workflow exceptions would still be thrown, they are not cleared with the clearErrorMessages

Reply:
oK, I shall merge